web-api/src/Kerbero.WebApi/Program.cs

```diff
@@ -12,11 +12,8 @@
 
 builder.Configuration.AddEnvironmentVariables(_ => Env.Load());
 
-builder.Host.ConfigureLogging(logging =>
-{
-	logging.ClearProviders();
-	logging.AddConsole();
-});
+builder.Logging.ClearProviders();
+builder.Logging.AddConsole();
```

Comment:
why? to it seems the same as the default logging setting of aspnetcore, so it should be removed